style.css

```diff
@@ -161,3 +161,335 @@ body {
+  flex: none;
+  flex-grow: 0;
+}
+
+.project-title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.project-role {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  flex: none;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.project-role img {
+  position: static;
+  width: 8px;
+  height: 8px;
+  left: 67px;
+  top: 12px;
+  background: #c1c7d0;
+  flex: none;
+  flex-grow: 0;
+  margin: 0 12px;
+}
+
+.project-role .client {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  flex-grow: 0;
+  margin: 0;
+}
+
+.project-role .role {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  flex-grow: 0;
+  margin: 0;
+}
+
+.project-role .year {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  flex-grow: 0;
+  margin: 0;
+}
+
+.description-text {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.tech-tags {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 8px;
+  margin-bottom: 20px;
+}
+
+.tech-tags-text {
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+}
+
+.tech-tag-name {
+  border-radius: 8px;
+  background-color: #ebebff;
+  padding: 4px 12px;
+}
+
+.btn-container {
+  width: 100%;
+  flex: none;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.view-project-btn {
+  display: flex;
+  align-items: center;
+  padding: 12px;
+  height: 48px;
+  background-color: #fff;
+  color: #6070ff;
+  border: 1px solid #7f8cff;
+  box-sizing: border-box;
+  text-decoration: none;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  transition-property: background-color, color;
+  transition-duration: 500ms;
+}
+
+.view-project-btn:enabled {
+  border: 1px solid #7f8cff;
+  box-sizing: border-box;
+}
+
+.view-project-btn:hover {
+  background-color: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+  cursor: pointer;
+}
+
+.view-project-btn p {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  flex: none;
+  flex-grow: 0;
+  margin: 0 10px;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 114px 24px;
+}
+
+.about-section {
+  display: flex;
+  flex-direction: column;
+  padding: 40% 24px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-top-right-radius: 80px;
+  background-color: #fff;
```

Comment:
- [ ] You did a good job adding the about me section but please give it a little more space between the about me section and skills section as you can see it in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1).
Here you have a screenshot of how you did it:
![image](https://user-images.githubusercontent.com/74506933/153710955-87abfdbc-5f23-4aa5-bf65-81ddb212d2d7.png)

- [ ] Please add the background of the about me section as you can see it in the design. All body has background-color: "#E5E5E5" so kindly fix it following the design.